projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-header/config-attribute-header.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { Configurator } from '@spartacus/core';
+import { ConfigUIKeyGeneratorService } from '../service/config-ui-key-generator.service';
+@Component({
+  selector: 'cx-config-attribute-header',
+  templateUrl: './config-attribute-header.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfigAttributeHeaderComponent {
+  constructor(private uiKeyGen: ConfigUIKeyGeneratorService) {}
+
+  @Input() attribute: Configurator.Attribute;
+
+  public showRequiredMessage(): boolean {
```

Comment:
FYI: In TypeScript, the class' members/methods are public by default, so no need for explicit `public`. It applies to whole PR. Not big deal though.

Reply:
done